tools/mage/js.go

```diff
@@ -169,6 +171,11 @@ func (js Js) Build() error {
 	if mg.Verbose() {
 		fmt.Println("Running Webpack")
 	}
+	isCI := os.Getenv("CI") == "true"
+	publicExists := pathExists("./public")
```

Comment:
This is a hack, which should not be there.
Please modify the dependencies accordingly instead of this.
Why would this be specific to CI?

Reply:
https://github.com/TheThingsNetwork/lorawan-stack/pull/2764#discussion_r447586088

Please acknowledge.